packages/react-ai/src/types.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Conversation as SDKConversation,
+  ConversationMessage as SDKConversationMessage,
+} from '@aws-amplify/data-schema/dist/esm/ai/ConversationType';
```

Comment:
Depending on exports in the dist folder feels fragile to me. Are they planning on officially exporting these types?

Reply:
We're still in discussion with the data team to see if they will support it. If they will they will export it directly, if not we will have to copy the types over ourselves